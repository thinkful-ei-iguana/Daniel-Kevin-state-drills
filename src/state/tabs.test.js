import React from 'react';
import ReactDOM from 'react-dom';
import Tabs from './Tabs';
import renderer from 'react-test-renderer'

describe(`Tabs Component`, () => {
  it('renders without errors', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Tabs />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
})

describe(`Tabs Component`, () => {
  // array of objects each with a name and content
  const tabsProp = [  {name: 'First tab',
  content: 'placeholder'},
  {name: 'Second tab',
  content: 'more placeholder'},
  {name: 'Third tab',
  content: 'even more placeholder'}]; 

  it('renders the first tab by default', () => {
    const tree = renderer.create(<Tabs tabs={tabsProp} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})