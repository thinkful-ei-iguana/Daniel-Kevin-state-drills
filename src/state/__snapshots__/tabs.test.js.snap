// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs Component renders the first tab by default 1`] = `
<div>
  0
</div>
`;
